projects/ng-storeon/src/lib/storeon.service.ts

```diff
@@ -39,7 +40,11 @@ export class StoreonService<State, Events extends StoreonEvents<State> = any> im
       );
     }
 
-    return mapped$.pipe(distinctUntilChanged());
+    return mapped.pipe(distinctUntilChanged());
+  }
+
+  clearStoreon() {
+    this.state.next(this.initialState);
```

Comment:
I really need some help on this.

Reply:
I don't think we need to create this method in the API